.zenodo.json

```diff
@@ -0,0 +1,103 @@
+{
+  "creators": [
+    {
+      "name": "Tal Yarkoni"
```

Comment:
An example update:

```suggestion
      "affiliation": "University of Texas at Austin",
      "name": "Tal Yarkoni",
      "orcid": "0000-0002-6558-5113"
```

Reply:
So do you want us to use `Last, First`, or not? ;-)

Reply:
I'll move them all before this goes in, though people can update theirs, if they like. The script is tolerant of both forms, now. (But it's really hard to automatically reorder to `Last, First`.)

Reply:
Note also that suggested diff shows only one line removed and 3 added... Is that just visualization glitch or how it would actually patch it?
Because of that I did my suggestions below over line at a time.
In the past I successfully provided multiple suggestions, may be GitHub broke them or something specific about it being a json or relatively large changes that it fails to correctly identify the context?

Reply:
You can only remove one at a time, but in this case, the other two fields don't exist.

Reply:
@tyarkoni Is this correct? I can go ahead and include it.